# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/lrzsz/prototypes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/lrz.c.orig	Sun Nov 28 11:04:25 1999
+++ ./src/lrz.c	Sun Nov 28 11:04:30 1999
@@ -41,12 +41,6 @@
 #include "xstrtoul.h"
 #include "error.h"
 
-#ifndef STRICT_PROTOTYPES
-extern time_t time();
-extern char *strerror();
-extern char *strstr();
-#endif
-
 #ifndef HAVE_ERRNO_DECLARATION
 extern int errno;
 #endif
--- ./src/lsz.c.orig	Sun Nov 28 11:04:50 1999
+++ ./src/lsz.c	Sun Nov 28 11:04:55 1999
@@ -50,12 +50,6 @@
 #include "xstrtoul.h"
 #include "error.h"
 
-#ifndef STRICT_PROTOTYPES
-extern time_t time();
-extern char *strerror();
-extern char *strstr();
-#endif
-
 #ifndef HAVE_ERRNO_DECLARATION
 extern int errno;
 #endif
